molr-commons/src/main/java/io/molr/commons/domain/dto/MissionRepresentationDto.java

```diff
@@ -21,26 +21,30 @@
     public final Set<BlockDto> blocks;
     public final Map<String, List<String>> childrenBlockIds;
     public final Set<String> breakpointBlockIds;
+    public final Set<String> ignoreBlockIds;
     
 
-    public MissionRepresentationDto(String rootBlockId, Set<BlockDto> blocks, Map<String, List<String>> childrenBlockIds, Set<String> breakpointBlockIds) {
+    public MissionRepresentationDto(String rootBlockId, Set<BlockDto> blocks, Map<String, List<String>> childrenBlockIds, Set<String> breakpointBlockIds, Set<String> ignoreBlockIds) {
         this.rootBlockId = rootBlockId;
         this.blocks = blocks;
         this.childrenBlockIds = childrenBlockIds;
         this.breakpointBlockIds = breakpointBlockIds;
+        this.ignoreBlockIds = ignoreBlockIds;
```

Comment:
Again: In a generalized form we should have here a multimap block->blockAttribute